applications/StructuralMechanicsApplication/tests/adjoint_sensitivity_analysis_tests/adjoint_beam_structure_3d2n/beam_test_local_stress_adjoint_parameters.json

```diff
@@ -36,9 +36,6 @@
             },
         "echo_level"                   : 0,
         "rotation_dofs"                      : true,
-        "linear_solver_settings"       : {
-            "solver_type"         : "ExternalSolversApplication.super_lu"
```

Comment:
I would still prescribe this like I did this in the `structural_mechanics_test_factory` I suggest to briefly discuss in person.